corehq/apps/app_manager/views/apps.py

```diff
@@ -392,6 +396,20 @@ def _inner(request, link_domain, data, master_domain=domain):
         return view_generic(request, domain, app_id=app_id, copy_app_form=form)
 
 
+@require_can_edit_apps
+def app_from_template(request, domain, slug):
+    send_hubspot_form(HUBSPOT_APP_TEMPLATE_FORM_ID, request)
+    clear_app_cache(request, domain)
+    template = load_app_template(slug)
+    app = import_app_util(template, domain, {
+        'created_from_template': '%s' % slug,
+    })
+    comment = _("A sample application you can try out in Web Apps")
+    build = make_async_build(app, request.user.username, release=True, comment=comment)
+    cloudcare_state = '{{"appId":"{}"}}'.format(build._id)
```

Comment:
Do the curly braces get encoded in the url?

Reply:
I don't think so